packages/libs/react-components/src/styles/stitches.config.ts

```diff
@@ -120,6 +126,7 @@ export const {
     shadows: {
       // TODO: Update to match design system
       1: `0px 1px 2px 0 $colors$neutral3`,
+      card: `0px 0px 10px rgba(${hexToRgb(colors.neutral1)}, .04)`,
```

Comment:
Generally speaking, we want to avoid using alpha colors unless they are really necessary. Would it be possible to achieve a similar result using a color without alpha?